gcloud/resource_manager/client.py

```diff
@@ -49,7 +49,7 @@ class Client(BaseClient):
     _connection_class = Connection
 
     def new_project(self, project_id, name=None, labels=None):
-        """Creates a :class:`.Project` bound to the current client.
+        """Create a :class:`.Project` bound to the current client.
```

Comment:
Repo owners need to decide whether english language usage is something you care about or not and consistently apply that decision. If you do, then consistent verb tense should be fair game.